packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -26,289 +26,464 @@
-            data-user="$ctrl.user"
-            data-version="$ctrl.model.engine.selectedVersion.version"
-        ></database-order-review>
-        <pci-order-command
-            data-ng-if="$ctrl.currentStep >= 5"
-            data-order-api-url="$ctrl.orderAPIUrl"
-            data-order-data="$ctrl.orderData"
-            data-user="$ctrl.user"
-        >
-            <p data-translate="pci_database_add_command_description"></p>
-            <a
-                class="oui-link_icon"
-                data-ng-href="{{:: $ctrl.apiGuideUrl }}"
-                target="_blank"
-                rel="noopener"
+
+            <!-- Step 6 -->
+            <div
+                data-ng-if="$ctrl.model.engine.selectedVersion.status === 'BETA'"
             >
-                <span
-                    data-translate="pci_database_add_command_documentation_link"
-                ></span>
-                <span
-                    class="oui-icon oui-icon-external-link"
-                    aria-hidden="true"
-                ></span>
-            </a>
-        </pci-order-command>
-        <div
-            class="mb-3"
-            data-ng-if="$ctrl.model.engine.selectedVersion.status === 'BETA'"
-        >
-            <pci-project-lab-agreements
-                data-lab="$ctrl.lab"
-                data-ng-if="!$ctrl.lab.isActivated()"
-                data-on-accept="$ctrl.acceptLab(accepted)"
-                data-on-contract-click="$ctrl.clickOnContract()"
-            ></pci-project-lab-agreements>
-        </div>
-        <div data-ng-if="$ctrl.processingOrder">
-            <oui-spinner data-size="s"></oui-spinner>
-            <span
-                data-translate="pci_database_add_create_database_processing"
-            ></span>
+                <h2>{{:: 'pci_database_add_step6_title' | translate }}</h2>
+                <div class="mb-3">
+                    <pci-project-lab-agreements
+                        data-lab="$ctrl.lab"
+                        data-ng-if="!$ctrl.lab.isActivated()"
+                        data-on-accept="$ctrl.acceptLab(accepted)"
+                        data-on-contract-click="$ctrl.clickOnContract()"
+                    ></pci-project-lab-agreements>
+                </div>
+            </div>
+        </form>
+    </div>
+    <div class="col-lg-3">
+        <div class="order-summary sticky-top mb-3">
+            <oui-tile
+                data-heading="{{:: 'pci_database_add_spec_heading' | translate }}"
+            >
+                <oui-tile-definition>
+                    <oui-tile-description>
+                        <div>
+                            <strong
+                                data-translate="pci_database_add_spec_name"
+                            ></strong
+                            ><span
+                                ng-bind="$ctrl.model.name"
+                                class="ml-2"
+                                oui-popover="lorem ipsum"
```

Comment:
Probably forget to have content ?

Reply:
Forgot to remove it, the `popover` is on the button next to the `span`, good catch ! https://github.com/ovh/manager/pull/6905/commits/80db90fb5024122d9219c182593f7dce517f26e9